grasping_benchmarks_ros/graspnet_ros/graspnet_grasp_planner_service.py

```diff
@@ -156,15 +157,22 @@ def read_images(self, req : GraspPlannerRequest):
 
         return camera_data
 
+    def read_aruco_board(self, req : GraspPlannerRequest):
+
+        aruco_board_data = self.create_aruco_board_data(req.aruco_board.position, req.aruco_board.orientation, req.grasp_filter_flag)
+   
+        return aruco_board_data
+
     def plan_grasp_handler(self, req : GraspPlannerRequest) -> GraspPlannerResponse:
 
         # Read camera images from the request
         camera_data = self.read_images(req)
+        aruco_board_data = self.read_aruco_board(req)        
```

Comment:
Why is this function called at all if there is no need to filter the grasp candidates?